src/components/Hero/styles.js

```diff
@@ -4,42 +4,46 @@ import { withStyles } from '@material-ui/core';
 const styles = theme => ({
   root: {
     backgroundColor: '#D34727',
-    padding: '20px 20px 100px',
+    padding: '0px 0px 80px',
```

Comment:
0 measurements do not need px, since 0px === 0rems === 0vw === 0em. I.e. its usually good practice to just write `0 0 80px`.

Reply:
Thanks, working on getting out of the habit of adding measurement units after 0.